lib/muffin/frostings/attributes.rb

```diff
@@ -63,5 +63,9 @@ class NestedAttribute
     def initialize(attributes)
       self.attributes = attributes
     end
+
+    def persisted?
+      attributes[:id].present?
+    end
```

Comment:
But `Attributes` is included in `Muffin::Base`, therefore it should already be defined (I think `persisted` should go there as well).